azurerm/internal/services/eventhub/eventhub_namespace_resource.go

```diff
@@ -199,6 +199,12 @@ func resourceArmEventHubNamespace() *schema.Resource {
 								},
 							},
 						},
+
+						"allow_trusted_microsoft_services_bypass_firewall": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
```

Comment:
This is risky and we need to confirm that this value is actually set to false when this resource is created. If it isn't false on creation, this will break every user using this resource because Terraform will try and update this value to false.

Reply:
it's false by default